### PR TITLE
Load .env file from DevContainer

### PR DESCRIPTION
**Pull request by thatnerdjosh**
Loads the .env file in the DevContainer environment as per instructions at: https://code.visualstudio.com/remote/advancedcontainers/environment-variables

### CODE REVIEW

**Comment by thatnerdjosh** [comment]
@ferronn-dev - I might have found a solution for this - testing now.

**Comment by thatnerdjosh** [comment]
Worked locally :crossed_fingers:

File: .devcontainer/devcontainer.json [hunk 1]
```diff
@@ -3,6 +3,7 @@
     "context": "..",
     "dockerfile": "Dockerfile"
   },
+  "initializeCommand": "cp .env.dist .env",
```
Comment (ferronn-dev) [inline .devcontainer/devcontainer.json:6]: Doesn't this obliterate any local changes?

Comment (thatnerdjosh) [reply .devcontainer/devcontainer.json:6]: Yup good point -- I'll update that to make it not do that

Comment (thatnerdjosh) [reply .devcontainer/devcontainer.json:6]: learned about a flag I didn't know about in the `cp` command... `-n` for no clobber :) tested locally and looks good just waiting for pre-commit checks

Comment (thatnerdjosh) [reply .devcontainer/devcontainer.json]: fixed/pushed

**Pull request merged**